components/Map.js

```diff
@@ -21,6 +21,28 @@ const public_works_streets_url =
 const city_council_districts_url =
   'https://services.arcgis.com/sFnw0xNflSi8J0uh/arcgis/rest/services/City_Council_Districts_View/FeatureServer/0';
 
+const walkable_streets_url =
+  'https://services.arcgis.com/sFnw0xNflSi8J0uh/arcgis/rest/services/BudgetFacilitiesFY2019/FeatureServer/7';
+
+const slow_streets_url =
```

Comment:
For future reference, JS style is for file-level constants to be CAPITAL_SNAKE_CASE.

Reply:
good to know - just updated